### PR TITLE
Home charts refactor

### LINKED ISSUE #372: Refactor charts
# Objetivos / Objectivos / Purpose

- Restructures the charts module to create a new template #367 .

### PR DESCRIPTION
**Pull request by esloch**
- Resolves #372 .

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,10 +25,13 @@ pip-selfcheck.json
 /shapely
 /share
 .ipynb_checkpoints
+notebooks_tmp/
 geodjango.db
 chunked_uploads
 authorized_keys
 
+AlertaDengue/static/geo_info.json
```
Comment (xmnlab) [inline .gitignore:33]: Is there anything inside static you need track? If not, you can igbore everthing inside static folder

File: AlertaDengue/dados/charts/cities.py [hunk 2]
```diff
@@ -8,7 +7,11 @@
 from plotly.subplots import make_subplots
 
 # local
-from .dbdata import get_series_by_UF, load_series
+from dados.dbdata import load_series
+
+'''
```
Comment (xmnlab) [inline AlertaDengue/dados/charts/cities.py:12]: if this is a module docstring it should be placed at the first line

File: AlertaDengue/dados/charts/home.py [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+from django.utils.translation import gettext as _
+
+import plotly.graph_objs as go
+import pandas as pd
+from plotly.subplots import make_subplots
+
+# local
+from dados.dbdata import get_series_by_UF
+
+'''
```
Comment (xmnlab) [inline AlertaDengue/dados/charts/home.py:12]: same here about module docstring

File: AlertaDengue/dados/charts/cities.py [hunk 4]
```diff
@@ -378,269 +381,9 @@ def create_tweet_chart(cls, df: pd.DataFrame, year_week):
...
-        return cls._create_chart(case_series, 'dengue')
-
-    @classmethod
-    def create_chik_chart(cls, case_series):
-        return cls._create_chart(case_series, 'chikungunya')
-
-    @classmethod
-    def create_zika_chart(cls, case_series):
-        return cls._create_chart(case_series, 'zika')
+'''
```
Comment (xmnlab) [inline AlertaDengue/dados/charts/cities.py:384]: this docstring looks out of place ...

File: AlertaDengue/dados/charts/states.py [hunk 5]
```diff
@@ -0,0 +1,105 @@
+import plotly.graph_objs as go
+import pandas as pd
+from plotly.subplots import make_subplots
+
+'''
```
Comment (xmnlab) [inline AlertaDengue/dados/charts/states.py:5]: same here about module's docstring

**Review by esloch — COMMENTED** [review]
Thanks @xmnlab !

**Pull request merged**